### PR TITLE
add randomBook Title

### PR DESCRIPTION
**Pull request by FarhadJahangiri**

### CODE REVIEW

File: random-book/src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// 		<main>
+// 			<p>{counter}</p>
+// 			<button onClick={() => setCounter(counter + 1)}>
+// 				Increment
+// 			</button>
+// 		</main>
+// 	);
+// };
+
+// ReactDOM.render(<App />, appDiv);
```
Comment (mbovel) [inline random-book/src/main.tsx]: You should probably remove this commented code.

File: random-book/src/main.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import * as ReactDOM from "react-dom";
+
+const appDiv = document.getElementById("app");
+
+if (!(appDiv instanceof HTMLDivElement)) {
+	throw new Error("No div with id 'app' found");
+}
+
+function RandomTitle(App) {
+    const [title, newTitle] = React.useState("Click For New Title");
```
Comment (mbovel) [inline random-book/src/main.tsx:11]: You can already display a random book title at the beginning.

File: random-book/src/main.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    const { titles } = App;
+    return (
+        <main>
+            <p>{title}</p>
```
Comment (mbovel) [inline random-book/src/main.tsx:16]: Should also display the book with a random background-color 😉

**Review by msom — APPROVED** [review]

**Comment by mbovel** [comment]
Note: @FarhadJahangiri , you only need to add @msom as reviewer for Python exercises 😄

File: random-book/src/main.tsx (hunk 1 above)
Comment (FarhadJahangiri) [reply random-book/src/main.tsx]: yes I kept it because it was exercise, i will remove it.

**Comment by FarhadJahangiri** [comment]
> 
> 
> Note: @FarhadJahangiri , you only need to add @msom as reviewer for Python exercises 😄

oh really !!! ok sorry @msom

File: random-book/src/main.tsx (hunk 3 above)
Comment (FarhadJahangiri) [reply random-book/src/main.tsx:16]: i will do it after doing second exercise.

Comment (mbovel) [reply random-book/src/main.tsx:16]: 👍

**Pull request merged**